src/main/java/io/gingersnapproject/cdc/configuration/Configuration.java

```diff
@@ -0,0 +1,17 @@
+package io.gingersnapproject.cdc.configuration;
+
+import java.util.Map;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+@ConfigMapping(prefix = "gingersnap")
+public interface Configuration {
+
+   Database database();
+
+   Cache cache();
```

Comment:
I am a bit confused with this `Cache` configuration here and the `Backend` configuration in the `Rule`. Shouldn't everything be together?

Also, to have cache + index would we need to have to deploy two instances?

Reply:
Something like `rule.transformation` should fit, but since we have stuff like columns and keys a +1 to collapse to the rule. But no problem to change it on another PR dedicated only to properties.

Reply:
It was a simple task so I've added it as an additional commit on this PR.